<launch>
  <node cwd="node" name="rosbag_node" pkg="rosbag" type="record"
	output="screen" args="-o $(find aauship_control) /imu /kf_attitude"/>

  <node cwd="node" name="KF_attitude_node" pkg="aauship_control" type="KF_attitude_node" output="screen"/>
</launch>
